### ISSUE TITLE
BUILD FAILED (OS X 12.3.1 using python-build 20180424)

### ISSUE THREAD

**Issue by mertcanaltin** [pyenv/pyenv#2343]
### hello my system is mac os Monterey 12.3.1 (21E258) when i install pyenv 3.8.5 i get this error my processor is intel i5

![Alt text](https://i.ibb.co/8b5H0hm/Ekran-Resmi-2022-04-28-23-03-43.png "a title")

**Comment by mertcanaltin** [comment]
SOLVED ! 

- Uninstall the old tools
 $` sudo rm -rf /Library/Developer/CommandLineTools`
 
- Reinstall Xcode Command line tools
$ `xcode-select --install`

- Install python using pyenv
 $ `pyenv install 3.8.5`

**Comment by native-api** [comment]
> SOLVED !
> 
>     * Uninstall the old tools
>       $` sudo rm -rf /Library/Developer/CommandLineTools`
> 
>     * Reinstall Xcode Command line tools
>       $ `xcode-select --install`
> 
>     * Install python using pyenv
>       $ `pyenv install 3.8.5`

Interesting! What does your `clang --version` show after the reinstall?

**Comment by mertcanaltin** [comment]
@native-api  
```Apple clang version 13.0.0 (clang-1300.0.27.3)
Target: x86_64-apple-darwin21.4.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

**Comment by oldwhite83** [comment]
```
pyenv --version
pyenv 1.2.20

```

```
gcc --version
Apple clang version 13.1.6 (clang-1316.0.21.2.5)
Target: x86_64-apple-darwin21.4.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
```

`BUILD FAILED (OS X 12.3.1 using python-build 20180424)`

My GCC version is 13.1.6, but I still report the same error

**Comment by joemillervi** [comment]
> SOLVED !
> 
> * Uninstall the old tools
>   $` sudo rm -rf /Library/Developer/CommandLineTools`
> * Reinstall Xcode Command line tools
>   $ `xcode-select --install`
> * Install python using pyenv
>   $ `pyenv install 3.8.5`

Reminder that this uses several gb and takes 5-10min

**Comment by araujohn** [comment]
I had to uninstall and reinstall Home Brew before it returned to work. It concerned the change from Mac Intel to Mac M1(Silicon). 
See the article below  from Josh Alletto to find out why. [https://earthly.dev/blog/homebrew-on-m1/#:~:text=On%20Intel%20Macs%2C%20Homebrew%2C%20and,%2Fusr%2Flocal%2Fbin%20.&text=Homebrew%20chose%20%2Fusr%2Flocal%2F,in%20your%20PATH%20by%20default.](URL)

**Comment by yxy92** [comment]
> I had to uninstall and reinstall Home Brew before it returned to work. It concerned the change from Mac Intel to Mac M1(Silicon).
> See the article below from Josh Alletto to find out why. [https://earthly.dev/blog/homebrew-on-m1/#:~:text=On%20Intel%20Macs%2C%20Homebrew%2C%20and,%2Fusr%2Flocal%2Fbin%20.&text=Homebrew%20chose%20%2Fusr%2Flocal%2F,in%20your%20PATH%20by%20default.](URL)

**Comment by lpozo** [comment]
> SOLVED !
> 
> * Uninstall the old tools
>   $` sudo rm -rf /Library/Developer/CommandLineTools`
> * Reinstall Xcode Command line tools
>   $ `xcode-select --install`
> * Install python using pyenv
>   $ `pyenv install 3.8.5`

Doesn't work for me. Still get the same error:

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/leodanispozoramos/.pyenv/versions/3.11.4/lib/python3.11/ssl.py", line 100, in <module>
    import _ssl             # if we can't import it, let the error propagate
    ^^^^^^^^^^^
ModuleNotFoundError: No module named '_ssl'
ERROR: The Python ssl extension was not compiled. Missing the OpenSSL lib?

Please consult to the Wiki page to fix the problem.
https://github.com/pyenv/pyenv/wiki/Common-build-problems


BUILD FAILED (OS X 13.5.1 using python-build 20180424)
```

**Comment by awenstrup** [comment]
I used Apple's migration assistant to setup my new M2 Mac (coming from Intel), and scoured the internet for over an hour. Ultimately, the reason python builds were failing on my machine was the presence of a few files

```
/usr/local/lib/libgettextlib.dylib
/usr/local/lib/libintl.dylib
/usr/local/include/libintl.h
```

Removing them solved my issue, see https://github.com/pyenv/pyenv/issues/1877 for details.

**Comment by yanhaijing** [comment]
> I used Apple's migration assistant to setup my new M2 Mac (coming from Intel), and scoured the internet for over an hour. Ultimately, the reason python builds were failing on my machine was the presence of a few files我使用 Apple 的迁移助手来设置我的新 M2 Mac（来自 Intel），并在互联网上搜索了一个多小时。最终，Python 构建在我的机器上失败的原因是存在一些文件
> 
> ```
> /usr/local/lib/libgettextlib.dylib
> /usr/local/lib/libintl.dylib
> /usr/local/include/libintl.h
> ```
> 
> Removing them solved my issue, see #1877 for details.删除它们解决了我的问题，有关详细信息，请参阅#1877。

Not work, my information.

Apple clang version 14.0.3 (clang-1403.0.22.14.1)
Target: arm64-apple-darwin22.3.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
